go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/kubernetes-csi/node-driver-registrar
 go 1.13
```

Comment:
Do we want to update go to 1.15 here too?

Reply:
Sure. we can keep it in this PR.. Added the same.